src/pages/components/code-snippet/code.mdx

```diff
@@ -1,42 +1,106 @@
 ---
 title: Code snippet
 description: Code snippets are small blocks of reusable code that can be inserted in a code file.
-tabs: ['Code', 'Usage', 'Style', 'Accessibility']
+tabs: ["Code", "Usage", "Style", "Accessibility"]
 ---
 
+import {
+  codeSnippet,
+  codeSnippetSingle,
+} from "../../../data/components/code-snippet.js";
+
 <ComponentDemo
```

Comment:
**Code snippet bug**
The code snippets themes aren't rendering correctly for `singe` and `multi-line`. Themes effected: Gray10, Gray 90, Gray100

What it's showing:
![image](https://user-images.githubusercontent.com/11670886/78405329-e2111980-75c5-11ea-8a51-f44cd5f32779.png)

What it should be:
![image](https://user-images.githubusercontent.com/11670886/78405434-1be22000-75c6-11ea-95b6-14a99ddfa8ba.png)